### PR TITLE
refactor: move/remove vendor styles

### PR DESCRIPTION
**Pull request by brian-smith-tcril**
this should cover most of https://github.com/openedx/frontend-app-library-authoring/issues/72

### CODE REVIEW

File: src/library-authoring/create-library/index.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
+@mixin line-height($fontSize: 16) {
+    line-height: $fontSize + px;
+    line-height: ($fontSize/10) + rem;
```
Comment (arbrandes) [inline src/library-authoring/create-library/index.scss:3]: Just checking: is this a polyfill?

File: src/library-authoring/list-libraries/LibraryListPage.jsx [hunk 2]
```diff
@@ -109,22 +109,22 @@ export class LibraryListPage extends React.Component {
 
     return (
       <>
-        <div className="wrapper-mast wrapper">
-          <header className="mast has-actions">
-            <ActionRow>
-              <h1 className="page-header">{intl.formatMessage(messages['library.list.page.heading'])}</h1>
-              <ActionRow.Spacer />
-              {libraries.count !== 0 && (
+        {libraries.count !== 0 && (
```
Comment (arbrandes) [inline src/library-authoring/list-libraries/LibraryListPage.jsx:112]: This is changing behavior, so it's better if we have separate commit for it with a corresponding `feat:` or `fix:`.

**Review by arbrandes — CHANGES_REQUESTED** [review]
Thanks @brian-smith-tcril!  What a relief to have `vendor/` gone!

I only had one question, and I think we should have a separate commit for the minor change in behavior.  Otherwise, I tested it and it looks great!

File: src/library-authoring/create-library/index.scss (hunk 1 above)
Comment (brian-smith-tcril) [reply src/library-authoring/create-library/index.scss:3]: i'm not sure, it was being used in this modal so i moved it from vendor to here

i'll gladly remove it and replace the usages with a simple `line-height: 28px` if you'd prefer

File: src/library-authoring/list-libraries/LibraryListPage.jsx (hunk 2 above)
Comment (brian-smith-tcril) [reply src/library-authoring/list-libraries/LibraryListPage.jsx:112]: https://github.com/openedx/frontend-app-library-authoring/pull/83/commits/3d787cc610f1c24b63b30c978208189b022d00f9

**Review by arbrandes — APPROVED** [review]
Works great!  Approved.

**Pull request merged**